### ISSUE TITLE
Support for nginx version 1.8 & 1.9

### ISSUE THREAD

**Issue by MitRandi** [opentracing-contrib/nginx-opentracing#131]
As you can see current nginx stable version 1.18.0 and mainline 1.19.0
https://www.nginx.com/blog/nginx-1-18-1-19-released/

At this moment latest version support by nginx-opentracing is 1.17.3
https://github.com/opentracing-contrib/nginx-opentracing/releases

**Comment by brian-villanueva** [comment]
Are there any known incompatibilities that would prevent usage with 1.18-1.19?

**Comment by bbrunod** [comment]
When I try to reload my nginx (version 1.8), I have this error :
```
nginx: [emerg] module "/etc/nginx/modules/ngx_http_opentracing_module.so" version 1017003 instead of 1018000 in /etc/nginx/nginx.conf:1
```

**Comment by rnburn** [comment]
Fixed in #139
